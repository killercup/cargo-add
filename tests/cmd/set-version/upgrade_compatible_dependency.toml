bin.name = "cargo-set-version"
args = ["set-version", "0.4.5", "-p", "cargo-list-test-fixture-dependency"]
status = "success"
stdout = ""
stderr = """
    Upgraded cargo-list-test-fixture-dependency from 0.4.3 to 0.4.5
Updated dependency cargo-list-test-fixture from 0.4.3 to 0.4.5
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
