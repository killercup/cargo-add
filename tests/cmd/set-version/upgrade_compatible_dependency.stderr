    Upgraded cargo-list-test-fixture-dependency from 0.4.3 to 0.4.5
Updated dependency cargo-list-test-fixture from 0.4.3 to 0.4.5
